zmq.hpp

```diff
@@ -2224,6 +2224,14 @@ class monitor_t
         on_monitor_started();
     }
 
+    operator void *() ZMQ_NOTHROW { return handle(); }
+
+    operator void const *() const ZMQ_NOTHROW { return handle(); }
```

Comment:
I'd rather not add these operators. `socket_t` and `context_t` have them because they are direct correspondents of the underlying C data structures. But that's not the case here, and it's not obvious that a cast to `void*` returns the monitor socket handle.

Reply:
Agreed.